apps/emqx_management/include/emqx_mgmt.hrl

```diff
@@ -14,6 +14,8 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
+-define(APP, emqx_management).
```

Comment:
Some other modules in this app define `APP` on their own and it's different from this one. So you can't just bring it here to avoid collisions/redefinitions.

Reply:
you're right.. :1st_place_medal: